system/core/language.php

```diff
@@ -33,4 +33,9 @@ function languageOptionList($select, $omit) {
+  
+  if(file_exists('system/languages/languages.json')){
+    $isoLangList = json_decode(file_get_contents('system/languages/languages.json'), true);
+  }else{
```

Comment:
Do we need this check and else? AFAICS we always have a `languages.json` whenever `languages.php` also exists, so we could just remove the if?

Reply:
This is resolved.